README.md

```diff
@@ -668,6 +671,8 @@ Here is a reference for all the built-in Paddle tasks available at the moment.
   - Configuration for the task was covered in the [`tasks.publish`](#publish) subsection.
 - `run$<id>`: runs a Python script or module.
   - Configuration for the task was covered in the [`tasks.run`](#run) subsection.
+  - You can provide extra arguments with `-PextraArgs=<args>` option. For
```

Comment:
And please, drop a line somewhere here that it is also still possible to pass arguments directly in paddle.yaml via `args` section in RunTask (it's missing somehow)